code/__HELPERS/see_through_maps.dm

```diff
@@ -0,0 +1,30 @@
+//global statics for the see_through_component coordinate maps
+//For ease of use, include a comment in the shape of the coordinate map, where O is nothing, X is a hidden tile and A is the object
+GLOBAL_LIST_INIT(see_through_maps, list(
+	// X
+	// A
+	"default" = list(
+						list(0, 1, 0)
+	),
+
+	// XXX
+	// XXX
+	// OXO
+	// OAO
+	"pan_shape" = list(
+		list(-1, 3, 0), list(0, 3, 0), list(1, 3, 0),
+		list(-1, 2, 0), list(0, 2, 0), list(1, 2, 0),
+					    list(0, 1, 0)
```

Comment:
~~why are z offsets an option~~

Reply:
The turf finding proc (TURF_FROM_COORDS_LIST) takes turf coordinates as list(x, y, z), so it's kind of a necessity thing? I could just stick it in at runtime if that's preferred